integ-test/src/test/java/org/opensearch/sql/ppl/StandaloneIT.java

```diff
@@ -98,6 +98,57 @@ public void testSourceFieldQuery() throws IOException {
         actual);
   }
 
+  @Test
+  public void testErrorsAcrossMultipleQueries() throws IOException {
```

Comment:
I think this is the wrong place. This `StandaloneIT` is for running SQL/PPL outside OpenSearch (standalone library mode). You may need to add a new one if no better place.

Reply:
Ok.